extensions/canvasMedical/v1/actions/createClaim/config/fields.ts

```diff
@@ -0,0 +1,94 @@
+import {
+  DateOnlySchema,
+  FieldType,
+  type Field,
+} from '@awell-health/extensions-core'
+import { z, type ZodTypeAny } from 'zod'
+import { JsonArraySchema, JsonSchema } from '../../../validation'
+
+// Note: https://docs.canvasmedical.com/reference/claim-create
+export const fields = {
+  status: {
+    id: 'status',
+    label: 'Status',
+    description:
+      'Accepts a value representing the status compliant with Canvas Medical.',
+    type: FieldType.STRING,
+    required: true,
+  },
+  type: {
+    id: 'type',
+    label: 'Type',
+    description:
+      'Accepts a JSON object representing the type compliant with Canvas Medical.',
+    type: FieldType.JSON,
+    required: true,
+  },
+  patientId: {
+    id: 'patientId',
+    label: 'Patient ID',
+    description: `A patient's unique ID.`,
```

Comment:
`The patient's unique resource ID (e.g. Patient/865058f6654149bd921264d91519af9e)`